.gitignore

```diff
@@ -9,6 +9,9 @@ temp/
 # Spring
 **/*packageOutputDirectory*
 
+# Sensitive files
```

Comment:
This is required to make sure no one checks in their encrypted environment variables.
There is an identical rule for .NET repo

Reply:
Do you really want to add this to the root of the repo?

Reply:
I guess so if you are copying what was done in the net repo

Reply:
yes :)